packages/swayze/lib/src/core/controller/table/header_state.dart

```diff
@@ -79,6 +81,13 @@ class SwayzeHeaderState {
     return result;
   })();
 
+  // TODO: [victor] probably a controller is a better solution
+  final bool dragging;
```

Comment:
As mentioned, probably this has to be in another place.

It adds too much responsibility for this state, which should not need to know if it is being dragged or not.